### ISSUE TITLE
Sample Vector Set Error

### ISSUE THREAD

**Issue by GlennUL** [usnistgov/ACVP-Server#242]
**environment**
Demo

**testSessionId**
361934

**vsId**
1461306

**Algorithm registration**
{
    "algorithms": [
        {
            "direction": [
                "encrypt",
                "decrypt"
            ],
            "keyLen": [
                128,
                256
            ],
            "algorithm": "ACVP-AES-ECB",
            "revision": "1.0"
        }
    ],
    "encryptAtRest": false,
    "isSample": true
}

**Expected behavior**
Expected to be able to download the sample vector and expected file.

**Additional context**
The sample vector and expected files have the following error inside the json files:

"error": "Cannot deserialize the current JSON object (e.g. {\"name\":\"value\"}) into type 'System.Collections.Generic.List`1[NIST.CVP.ACVTS.Libraries.Generation.AES_ECB.v1_0.TestCase]' because the type requires a JSON array (e.g. [1,2,3]) to deserialize correctly.\r\nTo fix this error either change the JSON to a JSON array (e.g. [1,2,3]) or change the deserialized type so that it is a normal .NET type (e.g. not a primitive type like integer, not a collection type like an array or List<T>) that can be deserialized from a JSON object. JsonObjectAttribute can also be added to the type to force it to deserialize from a JSON object.\r\nPath 'testGroups[20].tests.resultsArray', line 1, position 81798."

**Comment by livebe01** [comment]
Hi @GlennUL, I just checked the versions of the prompt and expected answers json from the server and they do not appear to contain the error you mention. Could this be an issue with your client? I see that you submitted the response for this vector set around 5:47 Eastern yesterday and the error you mention is the result of the server attempting to process your response file.

**Comment by GlennUL** [comment]
Hi @livebe01 Thanks for checking into this. I did submit a response file yesterday as you mentioned. Previous to that we had downloaded the vector without issue. When the test session came back as failing I downloaded the results and that is when I encountered the error across all the files. I have a meeting with our developer today and will also have him check things on our end to see if he can locate anything.
